share/lib/python/neuron/config.py

```diff
@@ -0,0 +1,18 @@
+def _parse_arguments(h):
+    """Map the C++ structure neuron::config::arguments into Python.
+
+    The Python version is accessible as neuron.config.arguments.
+    """
+    global arguments
+    arguments = {}
+    num_keys_double = h.nrn_num_config_keys()
+    num_keys = int(num_keys_double)
+    assert float(num_keys) == num_keys_double
+    for i in range(num_keys):
+        key = h.nrn_get_config_key(i)
+        val = h.nrn_get_config_val(i)
+        assert key not in arguments
+        if key == "NRN_ENABLE_MODEL_TESTS":
+            val = tuple(val.split(";"))
+        val = {"ON": True, "OFF": False}.get(val, val)
```

Comment:
Just remembered that we may not always have "ON", i.e.: https://github.com/neuronsimulator/nrntest/commit/2fbb3c8303c94d76113cc74f13ef9df0014fdf71

Reply:
Based on https://cmake.org/cmake/help/latest/command/if.html#basic-expressions we could add case-insensitive mappings for ON/YES/TRUE/Y to `True` and OFF/NO/FALSE/N to `False`, and convert to Python integer if possible?

Reply:
:+1:

Reply:
Done in 80f6a051d8e5acb575f938702a0ff9e71264f94c.